be/src/exec/pipeline/scan/olap_chunk_source.cpp

```diff
@@ -161,6 +161,7 @@ Status OlapChunkSource::_init_reader_params(const std::vector<std::unique_ptr<Ol
     _params.profile = _runtime_profile;
     _params.runtime_state = _runtime_state;
     _params.use_page_cache = _runtime_state->use_page_cache();
+    _params.use_pk_index = thrift_olap_scan_node.use_pk_index;
```

Comment:
Does it read the latest version data or the version send by FE?

Reply:
read the latest version, as described in design doc